version.txt

```diff
@@ -1 +1 @@
-v0.0.0-unreleased
\ No newline at end of file
+v0.0.0-test-arm64
```

Comment:
Is this change needed or can it be reverted?

Reply:
I believe it SHOULD be reverted :) I'll do it

Reply:
Done ;)